web/crux/src/app/agent/agent.service.ts

```diff
@@ -302,6 +305,7 @@ export default class AgentService {
     connection.status().subscribe(it => this.onAgentConnectionStatusChange(agent, it))
 
     this.logger.log(`Agent joined with id: ${request.id}, key: ${!!agent.publicKey}`)
+    this.agent_counter.inc()
```

Comment:
Does this decrease when an agent disconnects?

Reply:
Nope, this is a counter function that represents a single increasing counter whose value can only increase or be reset to zero on restart.